backend/models/reviews.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const comment = new mongoose.Schema({
+  lastReviewed: {
+    type: String
+  },
+  value: {
+    type: String
+  }
+});
+
+const section = new mongoose.Schema({
+  id: {
+    type: String
+  },
```

Comment:
is there a reason why this is different than `_id`? maybe naming can be clearer

Reply:
I will rename to sectionId as that is what is being stored here.